plot_data/colors.py

```diff
@@ -30,13 +32,13 @@ def from_hsv(cls, hue: float, saturation: float, value: float):
     def __str__(self):
         return f"rgb({round(self.red * 255)},{round(self.green * 255)},{round(self.blue * 255)})"
 
-    def to_dict(self, *args, **kwargs):  # SO weird
+    def to_dict(self, **_) -> JsonSerializable:
```

Comment:
Escaping kwargs with _ seems weird to me, but I cannot state if its an anti-pattern or not. In addition to that, I think that we should have all the super's to_dict arguments, even if we don't use them